recipes/aneris/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.2.0" %}
+{% set sha256 = "624412e196fcefc2708d472c6840509492f91327e1fabcaf46fe61ab9a0705ff" %}
+
+package:
+  name: aneris
+  version: {{ version }}
+
+source:
+  url: https://github.com/iiasa/aneris/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt
```

Comment:
I just did `"{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"` but again just because I think I was copying something, this should work too I guess.